src/services/s3/s3-router.ts

```diff
@@ -76,7 +89,7 @@ s3Router.get(
             const { userIds } = BatchResumeDownloadValidator.parse(req.body);
 
             const batchDownloadPromises = userIds.map((userId) =>
-                getResumeUrl(userId, s3)
+                getResumeUrl(userId, s3, BucketName.RP_2024_RESUMES)
```

Comment:
why does getResumeUrl depend on BuckerName.RESUMES?

Reply:
The original code passes the bucket name into GetObjectCommand() so just made another parameter for it